transformers4rec/torch/utils/examples_utils.py

```diff
@@ -90,3 +92,62 @@ def fit_and_evaluate(trainer, start_time_index, end_time_index, input_dir):
         trainer.wipe_memory()
 
     return aot_metrics
+
+
+class LabelSmoothCrossEntropyLoss(_WeightedLoss):
```

Comment:
What if we move `LabelSmoothCrossEntropyLoss` from `examples_utils.py` to a Python file deticated to losses. It will make it easier to find the documentation under e.g. 'Transformers4rec.torch.losses' and we can add the upcoming pairwise losses their too

Reply:
you're right ! I was not sure about whether we want to expose the losses module in this early version or not. I have just moved it to losses.py module.